content/guides/payment-terminals.md

```diff
@@ -0,0 +1,60 @@
+ on behalf of Merchants using a “merchant terminal” API key.
+
+## API Keys
+
+To create API keys, you first need to get in touch with Centrapay to be issued an Integrator Account and an “account owner” API key. An “account owner” is a special kind of role that allows you to manage your account.
+
+You can use this key to [Create an API Key](https://docs.centrapay.com/api/api-keys#create-api-key) with the “merchant terminal” role. A “merchant terminal” key has a role that can create, cancel, void and refund Payment Requests on behalf of merchants.
+
+> You are responsible for the safety of your API keys
+
+## Merchant Configs
+
+Centrapay [Merchant Configs](https://docs.centrapay.com/api/merchant-configs) represent an available set of configured payment methods that can be utilized by one or more payment terminals by a Merchant.
+
+Merchants and merchant configs are managed by Centrapay. The only details you need to be able to create payment requests on their behalf are the `merchantId` and `merchantConfigId` (aka `clientId`). Centrapay will send you these details through whatever channel works for you. Please get in touch to let us know.
+
+In the future Centrapay will allow an integrator to on-board their own merchants through APIs using integrator API keys.
+
+## Example Flows
+
+See [Payment Flows](https://docs.centrapay.com/guides/payment-flows) for an overview of the API calls required for different payment flows.
+
+## Terminal Interface Guidelines
+
+When configuring a terminal with Centrapay there are a few common touch points that require branded assets. Please use the [Centrapay Brand Assets](https://centrapay.com/brand-assets/) when building the UI for these screens.
+
+### Displaying a Button
+
+<img src="/merchant-terminal-display-button.png" alt="A merchant terminal displaying a Centrapay button in the middle." />
+
+Touch displays should show a button that loads the QR Code screen when pressed.
+
+### Displaying an Icon
+
+<img src="/merchant-terminal-display-icon.png" alt="A merchant terminal displaying a Centrapay icon in the bottom right corner." />
+
+Displays that don’t support touch should show an icon above the button used to load the QR Code screen.
+
+### Displaying QR Codes
+
+<img src="/merchant-terminal-display-button.png" alt="A merchant terminal displaying a Centrapay QR code." />
+
+Centrapay QR codes should be displayed with a Centrapay logo in the centre.
+
+## Integration Architecture
+
+We strongly recommend Centrapay APIs are invoked from your backend and not directly from your payment terminals. Centralizing the invocation of our APIs from your backend offers the following benefits:
+
+1. Merchant network administrators will not need to make additional firewall changes.
+2. Your Centrapay API Keys can be managed centrally inside your secured network.
+3. TLS negotiation with Centrapay APIs can be managed independently of terminal hardware and software updates.
```

Comment:
In the [Notion guide](https://www.notion.so/centrapay/Payment-Terminal-Guide-fa8dc4919b7d408094a7109ec87a9217), there's a section about mitigating network issues, and that is should be replaced with a link to the merchant integration error handling page. Let's add that in here :)

Reply:
![Screenshot 2023-02-10 at 4 21 07 PM](https://user-images.githubusercontent.com/32785419/217994105-fb40c5d0-6a2d-484b-87ac-437c906f5f8d.png)

Reply:
A small introductory paragraph would be nice :)

Reply:
Please refer to the conversation above. Thanks